File: objectModel/Java/src/main/java/com/microsoft/commondatamodel/objectmodel/cdm/CdmReferencesEntities.java
// Copyright (c) Microsoft Corporation.

package com.microsoft.commondatamodel.objectmodel.cdm;

import com.microsoft.commondatamodel.objectmodel.resolvedmodel.ResolvedEntityReferenceSet;
import com.microsoft.commondatamodel.objectmodel.utilities.ResolveOptions;

public interface CdmReferencesEntities {
  ResolvedEntityReferenceSet fetchResolvedEntityReferences(ResolveOptions resOpt);
}

File: objectModel/CSharp/Microsoft.CommonDataModel.ObjectModel/Cdm/CdmReferencesEntities.cs
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License. See License.txt in the project root for license information.

namespace Microsoft.CommonDataModel.ObjectModel.Cdm
{
    using Microsoft.CommonDataModel.ObjectModel.ResolvedModel;
    using Microsoft.CommonDataModel.ObjectModel.Utilities;
    using System;

    public interface CdmReferencesEntities
    {
        [Obsolete("For internal use only.")]
        ResolvedEntityReferenceSet FetchResolvedEntityReferences(ResolveOptions resOpt = null);
    }
}
